### DISCUSSION TITLE
CenterPopup 组件中使用 Steps，弹窗打开瞬间进度条会发生位移抖动

### DISCUSSION THREAD

**Discussion by MaxCDon** [ant-design/ant-design-mobile#5624] [Q&A]
代码：https://codesandbox.io/s/musing-montalcini-u256gb?file=/src/App.tsx

复现页面：https://u256gb.csb.app/

问题机型：安卓

https://user-images.githubusercontent.com/33407227/187602986-fc5e4f30-f1e3-4626-9aa9-4c3b38e99eb6.mov

**Comment by awmleer** [comment]
诶这个抖动我咋没看出来……？

**Reply by MaxCDon** [reply]
仔细看步骤条的小圆点，多看几次试试

**Reply by awmleer** [reply]
噢发现了，感觉可能和浏览器底层的渲染有关，比如逻辑像素取整了，不一定好解决，我先把它转成一个 issue

**Reply by awmleer** [reply]
#5673
